section_inverted_text_index.go

```diff
@@ -958,6 +976,11 @@ type invertedIndexOpaque struct {
 	numTermsPerPostingsList []int // key is postings list id
 	numLocsPerPostingsList  []int // key is postings list id
 
+	// store terms that are unnecessary for the term dictionaries but needed in doc values
+	// eg - encoded geoshapes
+	// docNum -> fieldID -> term
+	extraDocValues map[int]map[int][]byte
```

Comment:
Let's change the type of this map to ..
```
extraDocValues map[int]map[uint16][]byte
```

.. to avoid having to re-typecast `fieldID` back to an int in line 802 above.